lib/mayaUsd/render/vp2ShaderFragments/CMakeLists.txt

```diff
@@ -49,6 +49,18 @@ list(APPEND shaderFragments_xmls
     plugInfo.json
 )
 
+set(headers
+    shaderFragments.h
+)
+
+# -----------------------------------------------------------------------------
+# promote headers
+# -----------------------------------------------------------------------------
+mayaUsd_promoteHeaderList(HEADERS ${headers} SUBDIR render/vp2ShaderFragments)
```

Comment:
Private header shouldn't need to be promoted.

Reply:
Sorry @HamedSabri-adsk, I led you astray here. I reviewed these changes before I looked at those in #391, so I didn't see the usage of `shaderFragments.h` as a public header until I mentioned it here:
https://github.com/Autodesk/maya-usd/pull/391#discussion_r404337448

This actually is a public header. Since we're reaching across from `mayaUsd/nodes` to include a file from `mayaUsd/render`, `shaderFragments.h` should be both promoted **and** installed to make it public, meaning includes of it should use angle braces and the full path.